### PR TITLE
[RO-224] Consider timezone on update dates command

### PR DESCRIPTION
**Pull request by luisfn**
Ticket: https://oat-sa.atlassian.net/browse/RO-224

### Fixed
- Fixed command for update start and end dates when using timezone.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.5 - TBD
```
Comment (Skyld) [inline CHANGELOG.md:3]: It should be `2.0.6` I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
+### Fixed
+- Fixed command for update start and end dates when using timezone.
```
Comment (Skyld) [inline CHANGELOG.md:6]: Could you please rephrase this sentence? I think it's not really describing what was fixed and where.

File: docs/cli/modify-entity-line-item-change-dates-command.md [hunk 3]
```diff
@@ -31,12 +31,6 @@ For the full list of options please refer to the helper option:
 $ sudo -u www-data bin/console roster:modify-entity:line-item:change-dates -h
 ```
 
-## Related environment variables
-
-| Variable | Description |
-|----------|-------------|
-| `DATABASE_URL` | Database connection string. Supported formats are described [here](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url). |
-
```
Comment (Skyld) [inline docs/cli/modify-entity-line-item-change-dates-command.md:39]: Why did you remove this? Any particular reason?

File: docs/cli/modify-entity-line-item-change-dates-command.md [hunk 4]
```diff
@@ -46,6 +40,10 @@ $ sudo -u www-data bin/console roster:modify-entity:line-item:change-dates -i 1,
 ```shell script
 $ sudo -u www-data bin/console roster:modify-entity:line-item:change-dates -s slug1,slug2,slug3 --start-date 2020-01-01T00:00:00+0000 --end-date 2020-01-05T00:00:00+0000 --force
 ```
+- Updating dates in a different timezone (UTC+1)
+```shell script
+$ sudo -u www-data bin/console roster:modify-entity:line-item:change-dates -s slug1,slug2,slug3 --start-date 2020-01-01T00:00:00+0100 --end-date 2020-01-05T00:00:00+0100 --force
+```
```
Comment (Skyld) [inline docs/cli/modify-entity-line-item-change-dates-command.md:46]: Could you please add this also to the command's console help block?

**Review by Skyld — CHANGES_REQUESTED** [review]

File: docs/cli/modify-entity-line-item-change-dates-command.md (hunk 3 above)
Comment (luisfn) [reply docs/cli/modify-entity-line-item-change-dates-command.md:39]: Well, DATABASE_URL is related to the whole application, it was probably included here by mistake. Can't see why mention it here.

Comment (luisfn) [reply docs/cli/modify-entity-line-item-change-dates-command.md:39]: Will simply put it back for now, tought it was added by me by mistake, didn't saw it was already spread everywhere.

**Review by andreluizmachado — APPROVED** [review]
Great job @luisfn. I found nothing, just the changelog, I added a comment there.

**Review by andreluizmachado — CHANGES_REQUESTED** [review]

**Review by Skyld — APPROVED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes

**Review by andreluizmachado — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes (if applicable)
- [x] Documentation is updated according to changes (if applicable)

**Review by Skyld — APPROVED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes
